resources/windows/wix/customactions.wxs.erb

```diff
@@ -197,17 +197,56 @@ End If
 
     <!-- Due to PUP-6729, system may not have permission to modify DACL, so first take ownership -->
     <CustomAction
-        Id="SetTakeownPuppetPermissions"
-        Property="TakeownPuppetPermissions"
-        Value="&quot;[%WINDIR]\System32\cmd.exe&quot; /c IF EXIST &quot;[%ProgramData]\PuppetLabs\puppet&quot; &quot;[%WINDIR]\System32\takeown.exe&quot; /F &quot;[%ProgramData]\PuppetLabs\puppet\*&quot; /R /A /D N"
-        Execute="immediate"/>
-    <CustomAction
-        Id="TakeownPuppetPermissions"
-        BinaryKey="WixCA"
-        DllEntry="<%= @platform.architecture == "x64" ? "WixQuietExec64" : "WixQuietExec" %>"
-        Execute="deferred"
-        Return="check"
-        Impersonate="no"/>
+      Id="TakeownPuppetPermissions"
+      Script="vbscript"
+      Execute="deferred"
+      Return="check"
+      Impersonate="no">
+      <![CDATA[
+On Error Resume Next
```

Comment:
nit: It doesn't matter too much, but `Dim`s are only required if `Option Explicit` is set.

Reply:
Yeah, thanks... forgot that I was going to either add that in, or remove all of the `Dim`s